ibc-core/ics25-handler/Cargo.toml

```diff
@@ -81,3 +81,11 @@ parity-scale-codec = [
   "ibc-core-handler-types/parity-scale-codec",
   "ibc-primitives/parity-scale-codec",
 ]
+arbitrary = [
+  "ibc-core-client/arbitrary",
+  "ibc-core-connection/arbitrary",
+  "ibc-core-commitment-types/arbitrary",
```

Comment:
`ibc-core-channel/arbitrary` is missed. and router too.

Reply:
added `ibc-core-channel`

Reply:
Thanks for correctly adding `ibc-core-commitment-types/arbitrary`. It looks like, we forgot it for `parity-scale-codec` :sweat_smile: Can you please add it for us? (or give us permission to push commits to your branch)